tester_app/Gemfile

```diff
@@ -47,3 +47,12 @@ group :development do
   gem 'spring'
 end
 
+
+group :development do
+  gem 'rubocop', require: false
+  gem 'ruby-lint', require: false
+  gem 'scss_lint', require: false
```

Comment:
Maybe a comment:

// require: false is necessary for the linters as we only want them loaded when used by the linting rake tasks.